composer/trainer/trainer.py

```diff
@@ -120,7 +123,8 @@ def __init__(
             *,
             model: BaseMosaicModel,
             train_dataloader: Union[DataLoader, DataSpec],
-            eval_dataloader: Union[DataLoader, DataSpec],
+            eval_dataloader: Optional[Union[DataLoader, DataSpec]] = None,
+            evaluators: Optional[List[Evaluator]] = None,
```

Comment:
As discussed, let's unify `evaluators` and `eval_dataloaders` into a single `eval_dataloader` argument.